src.gen/@amzn/codewhisperer-streaming/tsconfig.json

```diff
@@ -7,8 +7,7 @@
     "removeComments": true,
     "resolveJsonModule": true,
     "rootDir": "src",
-    "useUnknownInCatchVariables": false,
-    "composite": true
+    "useUnknownInCatchVariables": false
   },
-  "exclude": ["test/", "dist-*", "node_modules"]
+  "exclude": ["test/"]
```

Comment:
This seems... strange? This was done by the generator?

Reply:
Yes, I haven't manually updated any file in this folder.